src/components/Header/index.tsx

```diff
@@ -71,13 +80,19 @@ export const Header = ({ auth, children, name }: HeaderProps) => {
                   setDisplayDropdown(!displayDropdown);
                 }}
               >
-                {user && user.name ? user.name : ''}
+                {userInfo.name || ''}
```

Comment:
I didn't realize that each time `authService.getUser()`, the SDK is making a XHR` request to get the user info. I think in a future ticket, we should find a way to persist it rather than having to make that request all the time.